android/src/main/java/com/adscientiam/capacitor/googlefit/GoogleFitPlugin.java

```diff
@@ -0,0 +1,292 @@
+package com.adscientiam.capacitor.googlefit;
```

Comment:
This file is duplicated, could you merge it with the other one?

Reply:
I removed one of these files